classes/console/console.gd

```diff
@@ -41,8 +41,8 @@ func run_command(cmd: String):
 				if path == "":
 					Singleton.log_msg("Scene does not exist.", Singleton.LogType.ERROR)
 				else:
-					Singleton.set_location = null
-					var err = Singleton.warp_to(path)
+					Singleton.warp_location = null
+					var err = Singleton.warp_to(path, $"/root/Main/Player")
```

Comment:
In future, if we go ahead with multiplayer and such, then we'd need to replace these absolute paths with some kind of reference, but this works.

Reply:
Oh yes, definitely.